### DISCUSSION TITLE
API endpoint sporadically returning error  'Failed to post handler success response. Http response code: 400'

### DISCUSSION THREAD

**Discussion by jakecollins1** [vercel/community#4090] [Help]
### Summary

I am seeing this error sporadically in our Sveltekit project when calling an api endpoint. Typically when seeing this error, retrying the api call succeeds. I am unable to reproduce locally.

`LAMBDA_RUNTIME Failed to post handler success response. Http response code: 400. RequestId: 5c8e4597-4cd7-4412-994d-0699d352c42a Error: Runtime exited without providing a reason Runtime.ExitError`

There are other issues I have seen such as [this](https://github.com/orgs/vercel/discussions/3248#discussion-5402727) where it is suggested that this could be caused by including a header with non-UTF-8 characters, particularly in the cookies.  We are not setting any cookies in our header, or have anything varying in our header that would cause a sporadic fail

### Steps to Reproduce

I cannot reproduce it locally, but the problem is occurring on a sveltekit API endpoint.

**Comment by amyegan** [comment]
Do you have a [minimal reproducible example](https://vercel.com/support/articles/creating-a-minimal-reproducible-example) that I can use to help with debugging?

**Reply by jakecollins1** [reply]
@amyegan I haven't been able to make a minimal reproducible example unfortunately, our app is large and complicated.  I do have a screenshot of cookie values at the time of the error being thrown, do any of these look like they would trigger a [400 Error?](https://vercel.com/docs/errors/application-error-codes#400:-malformed_request_header)
![screen_shot_2023-10-05_at_8 51 37_am](https://github.com/vercel/community/assets/11318603/32912476-4e31-4357-8e3c-4701a9789098)
